src/main/java/baseball/Controller/Controller.java

```diff
@@ -0,0 +1,23 @@
+package baseball.Controller;
+
+import baseball.entity.Computer;
+import baseball.service.Service;
+import baseball.view.Input;
+import baseball.view.Log;
+
+public class Controller {
+
+    private final Service service;
+
+    public Controller() {
+        service = new Service();
+    }

```

Comment:
어차피 기본 생성자에서 바로 주입해줄거면 필드에서 주입해주는게 나아보여요!
```suggestion
    private final Service service = new Service();
```